src/controllers/ProductReviewController.ts

```diff
@@ -0,0 +1,178 @@
+import { Request, Response } from 'express';
+import { db } from '../database/models';
+import { where } from 'sequelize';
+import validateProductReview from '../validations/validateProductReview';
+
+interface User {
+  userId: string;
+}
+
+interface CustomRequest extends Request {
+  user?: User;
+}
+interface Review {
+  reviewComment: string;
+  rating: number;
+}
+
+export default class ProductReviewController {
+  static async Review(req: CustomRequest, res: Response) {
+    validateProductReview(req.body as Review);
+    const user = (req as any).user;
+    try {
+      const foundUser = await db.User.findOne({
+        where: { userId: user.userId },
+      });
+
+      if (!foundUser) {
+        return res.status(404).json({
+          message: 'user with the given is not found',
```

Comment:
What do you mean by this message?

Reply:
the id word is missing

Reply:
Can you make it clear in the message because we can not know what you meant by "the given"